Sitemap/Url/GoogleImage.php

```diff
@@ -49,39 +49,116 @@ class GoogleImage
     /**
      * create a GoogleImage for your GoogleImageUrl
      *
-     * @param string      $loc
-     * @param string|null $caption      [optional]
-     * @param string|null $geo_location [optional]
-     * @param string|null $title        [optional]
-     * @param string|null $license      [optional]
+     * @param string      $location
+     * @param string|null $caption     [optional]
+     * @param string|null $geoLocation [optional]
+     * @param string|null $title       [optional]
+     * @param string|null $license     [optional]
      */
-    public function __construct($loc, $caption = null, $geo_location = null, $title = null, $license = null)
+    public function __construct($location, $caption = null, $geoLocation = null, $title = null, $license = null)
     {
-        $this->setLoc($loc);
+        $this->setLocation($location);
         $this->setCaption($caption);
-        $this->setGeoLocation($geo_location);
+        $this->setGeoLocation($geoLocation);
         $this->setTitle($title);
         $this->setLicense($license);
     }
 
+    public function __get(string $name)
```

Comment:
I've added a test that covers both `__get` and `__set` methods, that which you can read to understand why this is needed.
The idea is : as a developer using this bundle, if I've created a class that extends one of these URL classes, I can use these properties directly, because it is protected.
These magic methods (which I hate from all my soul) are only here to avoid BC in this particular case (now you understand why Symfony decided to go final by default ?).